modules/ROOT/pages/indexes/search-performance-indexes/managing-indexes.adoc

```diff
@@ -856,17 +856,17 @@ RETURN name, type, provider, options.indexConfig AS config, createStatement
-| "uniqueBookIsbn"                  | "RANGE"  | "range-1.0"        | {}                                                                                                                                                                                                                                                                                                                                                                                                                         | "CREATE CONSTRAINT `uniqueBookIsbn` FOR (n:`Book`) REQUIRE (n.`isbn`) IS UNIQUE OPTIONS {indexConfig: {}, indexProvider: 'range-1.0'}"                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| "rel_text_index_name"             | "TEXT"   | "text-2.0"         | {}                                                                                                                                                                                                                                                                                                                                                                                                                         | "CREATE TEXT INDEX `rel_text_index_name` FOR ()-[r:`KNOWS`]-() ON (r.`interest`)"                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| "uniqueBookIsbn"                  | "RANGE"  | "range-1.0"        | {}                                                                                                                                                                                                                                                                                                                                                                                                                         | "CREATE CONSTRAINT `uniqueBookIsbn` FOR (n:`Book`) REQUIRE (n.`isbn`) IS UNIQUE"                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
curious that this one still had OPTIONS 🤔 would have said that should already have been removed 🤷